plenum/test/client/test_protocol_version.py

```diff
@@ -2,58 +2,75 @@
 from plenum.common.constants import CURRENT_PROTOCOL_VERSION
 from plenum.common.exceptions import RequestNackedException
 from plenum.test.helper import sdk_send_signed_requests, \
-    sdk_get_and_check_replies, sdk_random_request_objects, sdk_sign_request_objects
+    sdk_get_and_check_replies, sdk_random_request_objects, \
+    sdk_sign_request_objects, sdk_get_bad_response
+
+error_msg = 'Please update libindy or indy-node to the latest stable version'
```

Comment:
Please add tests that we check for incorrect protocol versions for LedgerStatus in addition to Requests.

Reply:
ok